### PR TITLE
[TRUS-2064] - Un-map Trustees

### PR DESCRIPTION
**Pull request by jassalrichy**
Add TrusteeEmailYesNoPage for lead trustees, reorder display of user answers on printout

### CODE REVIEW

File: app/utils/print/playback/sections/LeadTrusteeBusiness.scala [hunk 1]
```diff
@@ -33,13 +32,14 @@ object LeadTrusteeBusiness {
       Some(Seq(AnswerSection(
         headingKey = Some(messages("answerPage.section.leadTrustee.subheading")),
         Seq(
-          stringQuestion(TrusteeOrgNamePage(index), userAnswers, "trusteeBusinessName"),
           yesNoQuestion(TrusteeUtrYesNoPage(index), userAnswers, "leadTrusteeUtrYesNo", name),
+          stringQuestion(TrusteeOrgNamePage(index), userAnswers, "trusteeBusinessName"),
```
Comment (jackheslop96) [inline app/utils/print/playback/sections/LeadTrusteeBusiness.scala]: TrusteeOrgName should be before TrusteeUtrYesNo

File: app/utils/print/playback/sections/LeadTrusteeBusiness.scala [hunk 2]
```diff
@@ -33,13 +32,14 @@ object LeadTrusteeBusiness {
       Some(Seq(AnswerSection(
         headingKey = Some(messages("answerPage.section.leadTrustee.subheading")),
         Seq(
-          stringQuestion(TrusteeOrgNamePage(index), userAnswers, "trusteeBusinessName"),
           yesNoQuestion(TrusteeUtrYesNoPage(index), userAnswers, "leadTrusteeUtrYesNo", name),
```
Comment (jackheslop96) [inline app/utils/print/playback/sections/LeadTrusteeBusiness.scala]: Where is the "Is this trustee a UK registered business?" yes/no question?

File: app/utils/print/playback/sections/LeadTrusteeIndividual.scala [hunk 3]
```diff
@@ -42,8 +42,9 @@ object LeadTrusteeIndividual {
             addressAnswers(index, userAnswers, countryOptions, name).flatten ++
             Seq(yesNoQuestion(TrusteePassportIDCardYesNoPage(index), userAnswers, "trusteePassportOrIdCardYesNo", name),
               passportOrIdCardQuestion(TrusteePassportIDCardPage(index), userAnswers, "trusteePassportOrIdCard", name, countryOptions),
```
Comment (jackheslop96) [inline app/utils/print/playback/sections/LeadTrusteeIndividual.scala]: Passport details should be before address as these are an alternative to NINO

File: app/utils/print/playback/sections/LeadTrusteeBusiness.scala (hunk 1 above)
Comment (jassalrichy) [reply app/utils/print/playback/sections/LeadTrusteeBusiness.scala]: As per Rachel's feed back:
Lead trustee - The 'Is trustee a UK registered company/' is the first question. 
The TrusteeUtrYesNo renders 'Is trustee a UK registered company?'

File: app/utils/print/playback/sections/LeadTrusteeBusiness.scala (hunk 2 above)
Comment (jassalrichy) [reply app/utils/print/playback/sections/LeadTrusteeBusiness.scala]: As per Rachel's feed back:
Lead trustee - The 'Is trustee a UK registered company/' is the first question.
The TrusteeUtrYesNo renders 'Is trustee a UK registered company?'

File: app/utils/print/playback/sections/LeadTrusteeIndividual.scala (hunk 3 above)
Comment (jassalrichy) [reply app/utils/print/playback/sections/LeadTrusteeIndividual.scala]: Applied.

**Comment by jassalrichy** [comment]
TrusteeAddressYesNoPage removed from extration and printout for lead trustees.

**Pull request merged**